docker/deployment/scripts/kubeSyncMongo.sh

```diff
@@ -1,41 +1,39 @@
 #!/bin/bash
 
 set -e
+set -x
 
-echo Mongo Dump
-ssh mongo-xf mongodump
+PRODCONTEXT=--context='languageforge'
```

Comment:
these will require anyone wanting to use this script will have to ensure their context names are exactly the same FYI.